stockflux-securities-master/src/components/InputForm.js

```diff
@@ -4,6 +4,8 @@ import { Link } from "react-router-dom";
 import "./InputForm.css";
 import { getSecurity } from "../services/SecuritiesService";
 import ErrorMessage from "./ErrorMessage";
+import TextField from "./reusableComponents/TextField";
```

Comment:
You can just stick these components in the same place as the others. No need for another folder. The name `reusableComponents` doesn't sound quite right to me

Reply:
Okay, sure. I've moved them.